lib/omnibus/packagers/rpm.rb

```diff
@@ -521,5 +521,29 @@ def safe_architecture
         Ohai['kernel']['machine']
       end
     end
+
+        #
+    # Install the specified packages
+    #
+    # @return [void]
+    #
+    def install(packages, enablerepo)
+      if null?(enablerepo)
```

Comment:
Not sure `null?` exists in ruby, should probably be `if enablerepo.nil?` instead